package-lock.json

```diff
@@ -0,0 +1,13623 @@
+{
+  "name": "clothes-market-server",
```

Comment:
Try to remove this file to avoid conflicts or try to install and update your dependencies according to the project's base requirements.